src/App.js

```diff
@@ -1,12 +1,90 @@
-
 import './App.css';
+import MovieList from './components/MovieList';
+import React,{useState} from "react";
+import {BrowserRouter as Router, Route, Switch} from 'react-router-dom';
+import Basic from './components/Review';
+import MovieDetail from './components/MovieDetail';
+import Filter from './components/Filter'
 
 function App() {
```

Comment:
please make it a class-based component, and you could simply create a branch from the assignment2 branch to have all the previous code.